src/core/absorption_chiller.jl

```diff
@@ -28,61 +28,217 @@
+
+    return absorp_chl
+end
+
+
+"""
+get_absorption_chiller_defaults(prime_mover::String, boiler_type::String, size_class::Int)
 
-        min_kw = min_ton * KWH_THERMAL_PER_TONHOUR
-        max_kw = max_ton * KWH_THERMAL_PER_TONHOUR
-        installed_cost_per_kw = installed_cost_per_ton / KWH_THERMAL_PER_TONHOUR
-        om_cost_per_kw = om_cost_per_ton / KWH_THERMAL_PER_TONHOUR
-
-        new(
-            min_ton,
-            max_ton,
-            cop_thermal,
-            cop_electric,
-            installed_cost_per_ton,
-            om_cost_per_ton,
-            macrs_option_years,
-            macrs_bonus_fraction,
-            min_kw,
-            max_kw,
-            installed_cost_per_kw,
-            om_cost_per_kw
+return a Dict{String, Any} by selecting the appropriate values from 
+data/chp/absorption_chiller_defaults.json, which contains values based on heat transfer medium (hot_water_or_steam)
+such as:
+- "installed_cost_per_ton"
+- "om_cost_per_ton"
+- "cop_thermal"
+- "tech_sizes_for_cost_curve"
+
+Unlike CHP, the AbsorptionChiller tech sizes inform a single rate for installed_cost_per_ton that uses max_ton as input;
+there is no piecewise linear cost curve for the AbsorptionChiller technology.
+
+Inputs: 
+max_ton::Float64 -- maximum size of the absorption chiller technology, in tons
+
+
+response keys and descriptions:
+"hot_water_or_steam" -- string indicator of heat transfer medium for absorption chiller
+"default_inputs" -- Dict{string, Float64} containing default values for absorption chiller technology (see above)
+"""
+function get_absorption_chiller_defaults(;
+    max_ton::Float64 = 0.0, 
+    hot_water_or_steam::Union{String, Nothing} = nothing, 
+    chp_prime_mover::Union{String, Nothing} = nothing,
+    existing_boiler::Union{ExistingBoiler, Nothing} = nothing
+    )
+    acds = JSON.parsefile(joinpath(dirname(@__FILE__), "..", "..", "data", "absorption_chiller", "absorption_chiller_defaults.json"))
+    htf_defaults = Dict{String, Any}()
+
+    # convert Vector{Any} to Vector{Float64}
+    for htf in heat_transfer_mediums
+        for (k, v) in acds[htf]
+            if typeof(v) <: AbstractVector{Any}
+                acds[htf][k] = convert(Vector{Float64}, v)  # JSON.parsefile makes things Vector{Any}
+            end
+        end
+    end
+
+    #check required inputs
+    if isnothing(hot_water_or_steam)
+        if !isnothing(chp_prime_mover)
+            if chp_prime_mover == "combustion_engine"
+                hot_water_or_steam = "steam"
+            else  #if chp_prime mover is blank or is anything but "combustion engine" then assume hot water
+                hot_water_or_steam = "hot_water"
+            end
+        elseif !isnothing(existing_boiler)
+            hot_water_or_steam = existing_boiler.production_type
+        else
+            @error "Invalid argument for `hot_water_or_steam`; must be `hot_water` or `steam`"
```

Comment:
Let's default to "hot_water" if no other data is given to inform this.

Reply:
Makes sense to me!  Implemented in cd21cee4bbae3f40a6b1dbe60ac65d79240ea9f5